frontend/public/components/factory/details.tsx

```diff
@@ -113,6 +114,7 @@ export type DetailsPageProps = {
   name?: string;
   namespace?: string;
   resources?: FirehoseResource[];
+  breadcrumbs?: { name: string; path: string }[];
   breadcrumbsFor?: (obj: K8sResourceKind) => { name: string; path: string }[];
```

Comment:
We shouldn't have two props that are essentially the same. I would just leave `breadcrumbsFor` and callers can always use `breadCrumbsFor={() => breadcrumbs}` if needed.

Reply:
ok, reverting it back...

Reply:
Reverted to single `breadcrumbsFor` prop., made `breadcrumbsForGlobalConfig` return array of breadcrumbs, code clean up for `breadcrumbsForDetailsPage`